db/docs/source/introduction.rst

```diff
@@ -62,6 +62,26 @@ The source imagery is linked via the ``capture_source_id`` attribute of building
 The NZ Aerial Surveys data dictionary is here: https://nz-imagery-surveys.readthedocs.io/en/latest/index.html
 
 
+Locality
+--------
+Locality includes suburb, locality, island and park reserve. Water features such as coastal bay, lake and inland bay are not included.
+
+For building outline which intersects with two localities, the locality with larger area of intersection is used for that feature.
+
+.. figure:: _static/locality_larger_area.png
+   :scale: 70%
+   :alt: Examples of locality
+
```

Comment:
Maybe scale the images slightly smaller - they don't have to be too big because the image does require much detail to be helpful in explaining the topic. Try 40, 50 or 60%